input/fsh/SD_Genomics.fsh

```diff
@@ -110,9 +110,8 @@ Observation resources associated with an mCODE patient with an Observation.code
 
 * status and code and subject and effective[x] and value[x] and focus MS
 * subject 1..1
-* code from TumorMarkerTestVS (extensible)
+* code from TumorMarkerTestVS (required)
 * subject only Reference(CancerPatient)
-* focus only Reference(CancerConditionParent)
 * effective[x] only dateTime or Period
```

Comment:
```suggestion
* focus ^short = "What the tumor marker test is about when it's not applicable to the patient in general."
* focus ^definition = "What the tumor marker test is about when it's not applicable to the patient in general. For example, if the tumor marker test is from blood or urine, `focus` should be left empty. If the tumor marker test involves a specific [Tumor](StructureDefinition-mcode-tumor.html) or [Primary](StructureDefinition-mcode-primary-cancer-condition.html)/[Secondary](StructureDefinition-mcode-secondary-cancer-condition.html) Cancer Condition, reference a resource conforming to those profiles here."
* effective[x] only dateTime or Period
```

Reply:
Still not convinced. If there is a specimen involved, then the specimen would be the "focus". Otherwise, these are systemic tests, I believe.